docs/source/usersguide/install.rst

```diff
@@ -416,7 +416,9 @@ Prerequisites
 The Python API works with Python 3.4+. In addition to Python itself, the API
 relies on a number of third-party packages. All prerequisites can be installed
 using Conda_ (recommended), pip_, or through the package manager in most Linux
-distributions.
+distributions. To run simulations in parallel using MPI, it is recommended to
+build mpi4py, hdf5, h5py from source, in that order, using the same compilers
```

Comment:
HDF5 should be capitalized. Also, there should be no strict ordering for the most part. The only requirement is that h5py is built off an existing HDF5 installation.

Reply:
To use MPI, I think there are the following ordering : 
mpi4py is required for h5py
HDF5 is required for h5py
Because I have had trouble making h5py 2.7.1 work with mpi4py 3.0.0, I think stating the dependencies among the "optional" libraries can help people install openMC.